nucliadb/src/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -114,7 +114,9 @@ async def create(
 
         try:
             async with driver.transaction() as txn:
-                exists = await datamanagers.kb.get_kb_uuid(txn, slug=slug)
+                exists = await datamanagers.kb.get_kb_uuid(
+                    txn, slug=slug
+                ) or await datamanagers.kb.exists_kb(txn, kbid=kbid)
```

Comment:
shouldn't it simply be `exists = await datamanagers.kb.exists_kb(txn, kbid=kbid)` ?